Typechain module results in `'PromiseOrValue' is declared but never used.`

I'm going to look into this more, but building the SDK does not seem to work anymore. As far as I can tell it's an issue with "@typechain/ethers-v5" that came out of changes moving from "10.0.0" to "10.1.0".
There's an issue open here: https://github.com/dethcrypto/TypeChain/issues/721 but not much conversation.
If we pin at "10.0.0" other dependencies have a conflict and we get warnings ala `npm WARN ERESOLVE overriding peer dependency`.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@tableland/evm",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "description": "Tableland Tables EVM contracts and client components",
   "engines": {
     "node": ">=14.0.0"

--- a/tsconfig.build.json
+++ b/tsconfig.build.json
@@ -5,7 +5,8 @@
     "outDir": null,
     "declaration": true,
     "sourceMap": true,
-    "declarationMap": true
+    "declarationMap": true,
+    "skipLibCheck": true
   },
   "include": ["./typechain-types/**/*"],
   "files": ["proxies.ts"]

